### PR TITLE
Update default distribution versions

### PR DESCRIPTION
**Pull request by wolf31o2**
Switch to the latest versions supported by caskdata/cdap (it pays the bills) out of the box. :smiley:

### CODE REVIEW

**Comment by wolf31o2** [comment]
Waiting on this... Need to update tests due to changes in default versions. Going to be explicit about versions tested on every context.

File: recipes/hive.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Cookbook Name:: hadoop
 # Recipe:: hive
 #
-# Copyright © 2013-2014 Cask Data, Inc.
+# Copyright © 2013-2016 Cask Data, Inc.
```
Comment (wolf31o2) [inline recipes/hive.rb]: This file was updated in #251 but the copyright wasn't updated.

**Comment by wolf31o2** [comment]
This is now ready for review.

**Comment by dereklwood** [comment]
👍 lgtm.  might need to update again once we validate hdp 2.4

**Pull request merged**